be/tool/beck.c

```diff
@@ -1368,10 +1368,12 @@ static struct m0_stob_ad_domain *emap_dom_find(const struct action *act,
 
 	for (i = 0; i < act->a_builder->b_ad_dom_count; i++) {
 		adom = act->a_builder->b_ad_domain[i];
+#ifdef NEW_BTREE_INTEGRATION_COMPLETE
```

Comment:
Is this check still needed ?

Reply:
Yes we have kept it to make sure the code compiles. At some point in time we will try to make changes in the beck tool to work with the new Btree.